src/components/navigation-bar/navigation-bar.component.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AppWrapper from '../app-wrapper/app-wrapper';
+import StyledNavigationBar from './navigation-bar.style';
+
+const NavigationBar = ({ navigationType, children, ariaLabel }) => {
+  return (
+    <StyledNavigationBar
+      role='navigation'
+      aria-label={ ariaLabel }
+      navigationType={ navigationType }
+      data-component='navigation-bar'
+    >
+      <AppWrapper className='carbon-navigation-bar__content'>
+        { children }
+      </AppWrapper>
+    </StyledNavigationBar>
+  );
+};
+
+NavigationBar.propTypes = {
+  children: PropTypes.node,
+  ariaLabel: PropTypes.string,
+  navigationType: PropTypes.oneOf(['light', 'dark'])
+};
+
+NavigationBar.defaultProps = {
```

Comment:
`defaultProps` is being deprecated for functional components - https://github.com/reactjs/rfcs/blob/3416f0de38ee120f6d9dac7f7c9405b77cf37bdc/text/0000-create-element-changes.md#deprecate-defaultprops-on-function-components 
So best to add the default as `navigationType = "light"` up on line 6

Reply:
You are right, I'm happy to follow this one :)

Reply:
I will also use default props in `style.js` file

Reply:
My bad :D It won't work in the `style.js` Will just add this to the main component